portal-bridge/src/bridge/state.rs

```diff
@@ -47,6 +47,10 @@ pub struct StateBridge {
     census_tx: mpsc::UnboundedSender<EnrsRequest>,
     // Global offer report for tallying total performance of state bridge
     global_offer_report: Arc<Mutex<GlobalOfferReport>>,
+    // Bridge id used to determine which content keys to gossip
+    bridge_id: u64,
```

Comment:
Why not make this: `bridge_id: BridgeId`?

You can also move `should_gossip_content(&self, content_idx: usize) -> bool` inside BridgeId. But maybe rename it to `is_selected(&self, index:usize) -> bool`. Up to you?